### PR TITLE
Unbreak Travis and unskip some tests

### PR DESCRIPTION
**Pull request by anurse**
There's a WebSocket client for Linux and macOS! 🎉 . Ok, so maybe it was there a long time ago, but these tests never got unskipped 🤦‍♂️ 

Also, Travis was broken because of wstest issues. I made that more robust by using [`virtualenv`](https://virtualenv.pypa.io/en/stable/)
  
There's an indentation fix in this PR as well, so I suggest reviewing with [whitespace ignored](https://github.com/aspnet/WebSockets/pull/219/files?w=1)

### CODE REVIEW

File: test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs [hunk 1]
```diff
@@ -319,223 +306,211 @@ public class WebSocketMiddlewareTests
...
-    [ConditionalFact]
-    [OSSkipCondition(OperatingSystems.Linux, SkipReason = "No WebSockets Client for this platform")]
-    [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "No WebSockets Client for this platform")]
-    [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, SkipReason = "No WebSockets Client for this platform")]
-    public async Task ReceiveFragmentedData_Success()
-    {
-        var orriginalData = Encoding.UTF8.GetBytes("Hello World");
-        using (var server = KestrelWebSocketHelpers.CreateServer(async context =>
+        [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, SkipReason = "No WebSockets Client for this platform")]
```
Comment (Tratcher) [inline test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs]: No Win7 yet?

**Comment by jkotalik** [comment]
Are autobahn tests supposed to run on AppVeyor?

**Comment by anurse** [comment]
I believe I gave up on supporting them when I first attempted it. Getting Python installed and running correctly was a pain.

File: test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs (hunk 1 above)
Comment (anurse) [reply test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs]: I don't believe so.

Comment (anurse) [reply test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs]: Certainly not in full framework.

**Pull request merged**